### PR TITLE
Added missing translations

### PR DESCRIPTION
**Pull request by sebeder**
https://trello.com/c/3hPV4hhb/343-eigenschaften-bei-ceg-knoten-fehlende-%C3%BCbersetzung

### CODE REVIEW

**Comment by bliemelh** [comment]
uebersetzung

The translations work. I don't know, if the descriptive text underneath should be translated aswell.

**Comment by bliemelh** [comment]
expectedoutcome

Would be nice if the "expected outcome" is translated, too, for a node in the process model.

**Comment by sebeder** [comment]
Good morning @bliemelh , I just added translations for the field descriptions as well. Can you have a look at it again?

**Comment by bliemelh** [comment]
Hi @sebeder , works fine, but Expected Outcome is still not translated. Check field-meta.ts. shortDesc has the value 'Expected Outcome', should have 'ExpectedOutcome' without the blank space in order to be translated.

**Comment by sebeder** [comment]
Hi @bliemelh , I just added the missing translation. Thank you!

File: bundles/specmate-config/config/specmate-config.properties [hunk 1]
```diff
@@ -53,18 +53,6 @@ search.maxResults 		= 100
...
-project.jira.connector.pid           = com.specmate.connectors.jira.JiraConnector
-project.jira.connector.jira.url      = https://qualicen.atlassian.net
-project.jira.connector.jira.project  = SPEM
-project.jira.connector.jira.username = maximilian.junker@qualicen.de
-project.jira.connector.jira.password = ***REMOVED***
-project.jira.connector.connectorID   = jira
-
-
-
 # User session
```
Comment (bliemelh) [inline bundles/specmate-config/config/specmate-config.properties]: Not sure what this is about

**Comment by bliemelh** [comment]
Looks good now, I'm just not sure what specmate-config.properties is about.

**Pull request merged**